Cargo.toml

```diff
@@ -124,6 +124,7 @@ include = [
     "Cargo.toml",
     "LICENSE",
     "README.md",
+    "*.rs",
```

Comment:
let's just make this `build.rs`, I'd rather not blanket-include things outside `src`